### PR TITLE
chore(ui): TE-1597 hide alerts after 10 seconds

### PR DESCRIPTION
**Pull request by jacksonargo**
* Alerts automatically go away after 10 seconds.
* First alert is displayed after 0.5s instead of 0.1s. (Gives enough time for an api request to complete before showing 'authentication has been disabled' 😩 )
* ~Only the outermost exception is reported, instead of recursively through the error stack.~

https://github.com/startreedata/thirdeye/assets/7391437/eef1fe18-425b-4615-8bb8-dbd1d0def610

### CODE REVIEW

File: thirdeye-server/src/main/java/ai/startree/thirdeye/core/ExceptionHandler.java [hunk 1]
```diff
@@ -92,9 +92,5 @@ private static void populateStatusListApi(final Throwable e, StatusListApi statu
         break;
       }
     }
-
-    if (e.getCause() != null) {
```
Comment (suvodeep-pyne) [inline thirdeye-server/src/main/java/ai/startree/thirdeye/core/ExceptionHandler.java:96]: This actually helps understand the cause. Else, we have to see the logs. We don't have an alternate mechanism

File: thirdeye-ui/src/app/platform/components/notification-provider-v1/notification-display-v1/notification-display-v1.component.tsx [hunk 2]
```diff
@@ -65,6 +65,8 @@ export const NotificationDisplayV1: FunctionComponent<NotificationDisplayV1Props
                     <Snackbar
                         open
                         anchorOrigin={{ vertical: "top", horizontal: "center" }}
+                        autoHideDuration={3000}
```
Comment (suvodeep-pyne) [inline thirdeye-ui/src/app/platform/components/notification-provider-v1/notification-display-v1/notification-display-v1.component.tsx:68]: What is the behavior if you have a long list and have them timeout? Also, I think having a 10s timeout would be better. Some of the msg are long and 3s is too short. thoughts?

**Review by suvodeep-pyne — CHANGES_REQUESTED** [review]
Great initiative @jacksonargo in driving this.

File: thirdeye-ui/src/app/platform/components/notification-provider-v1/notification-display-v1/notification-display-v1.component.tsx (hunk 2 above)
Comment (cyrilou242) [reply thirdeye-ui/src/app/platform/components/notification-provider-v1/notification-display-v1/notification-display-v1.component.tsx:68]: +1, even 20s is more reasonable to start with I think.

Comment (jacksonargo) [reply thirdeye-ui/src/app/platform/components/notification-provider-v1/notification-display-v1/notification-display-v1.component.tsx:68]: Ok ya I'll do 10s. The timeout is for each alert serially, so if there are 20 alerts, then first one displays and timeouts out, then next displays and so on.

Comment (suvodeep-pyne) [reply thirdeye-ui/src/app/platform/components/notification-provider-v1/notification-display-v1/notification-display-v1.component.tsx:68]: thanks for sharing the compounding thing. 10s looks ok to me. We will do a longer term thing when @spham92  is back.

File: thirdeye-server/src/main/java/ai/startree/thirdeye/core/ExceptionHandler.java (hunk 1 above)
Comment (jacksonargo) [reply thirdeye-server/src/main/java/ai/startree/thirdeye/core/ExceptionHandler.java:96]: Ok I'll leave them in. I was going to do one big ui alert that contains all the messages, but the display looks messy so I'll leave it as is.

**Pull request merged**